ui/apps/platform/src/Containers/Vulnerabilities/NodeCves/Node/NodePageHeader.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Flex, Skeleton, Title, LabelGroup, Label } from '@patternfly/react-core';
+
+import { gql } from '@apollo/client';
+import { getDateTime } from 'utils/dateUtils';
+
+export const nodeMetadataFragment = gql`
+    fragment NodeMetadata on Node {
+        id
+        name
+        operatingSystem
+        kubeletVersion
+        kernelVersion
+        scanTime
+    }
+`;
+
+export type NodeMetadata = {
+    id: string;
+    name: string;
+    operatingSystem: string;
+    kubeletVersion: string;
+    kernelVersion: string;
+    scanTime?: string;
+};
+
+export type NodePageHeaderProps = {
+    data: NodeMetadata | undefined;
+};
+
+function NodePageHeader({ data }: NodePageHeaderProps) {
+    if (!data) {
+        return (
+            <Flex
+                direction={{ default: 'column' }}
+                spaceItems={{ default: 'spaceItemsXs' }}
+                className="pf-u-w-50"
+            >
+                <Skeleton screenreaderText="Loading Node name" fontSize="2xl" />
+                <Skeleton screenreaderText="Loading Node metadata" height="40px" />
+            </Flex>
+        );
+    }
+
+    const numLabels = data.scanTime ? 4 : 3;
+
+    return (
+        <Flex direction={{ default: 'column' }} alignItems={{ default: 'alignItemsFlexStart' }}>
+            <Title headingLevel="h1" className="pf-u-mb-sm">
+                {data.name}
+            </Title>
+            <LabelGroup numLabels={numLabels}>
+                <Label>OS: {data.operatingSystem}</Label>
+                <Label>Kubelet: {data.kubeletVersion}</Label>
+                <Label>Kernel version: {data.kernelVersion}</Label>
+                {data.scanTime && <Label>Scan time: {getDateTime(data.scanTime)}</Label>}
+            </LabelGroup>
+        </Flex>
+    );
+}
+
+export default NodePageHeader;
```

Comment:
@bradr5 Dave's recent PRs may give you some examples of how to create the components for Compliance. Maybe after we have a few different `__PageHeader` components, we can see how to make them more generic for reuse.

Reply:
For reference, we do have a semi-generic [component]([url](https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Vulnerabilities/components/CvePageHeader.tsx)) for CVE page headers across Workload/Node/Platform. It might be useful to abstract that to all page headers, and pass Labels, etc. as `children`.

We'd have to weight the balance between having a header layout component with unique children versus more header components that are specific to the page context.